.github/workflows/ui-tests.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         uses: actions/upload-artifact@v4
         if: ${{  failure() || cancelled() }}
         with:
-          name: raw-swiftui-test-output-${{matrix.platform}}-xcode-${{matrix.xcode}}-os-${{matrix.test-destination-os}}
+          name: raw-ios-swiftui-test-output-xcode-${{matrix.xcode}}-${{matrix.device}}
```

Comment:
This one was copypasta'd from the unit tests. There is no `platform` or `test-destination-os` in the matrix for this job.